README.md

```diff
@@ -38,13 +38,13 @@ Maven:
 <dependency>
     <groupId>uk.gov.hmcts.reform</groupId>
     <artifactId>java-logging-spring</artifactId>
-    <version>1.5.0</version>
+    <version>1.5.1</version>
```

Comment:
1.6.0?
seems like a new feature

Reply:
extension of a feature. i keep on diminishing these kinda small additions to bugfix semver upgrade.

still want 1.6.0 though? just checking

Reply:
I would prefer 1.6.0 its new functionality for sure.
but up to you